src/components/vm/consoles/consoles.jsx

```diff
@@ -213,15 +213,19 @@ export const ConsoleCard = ({ state, vm, config, onAddErrorNotification, isExpan
         );
     }
 
+    const urlOptions = { name: vm.name, connection: vm.connectionName };
+    const path = isExpanded ? ["vm"] : ["vm", "console"];
+    const href = cockpit.location.encode(path, { ...cockpit.location.options, ...urlOptions });
+    const name_parts = window.name.split("/");
+    const parent_pathname = name_parts.length > 0 ? "/" + name_parts[name_parts.length - 1] : "";
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2249-f787e1e8-20250729-135355-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)